critiquebrainz/frontend/templates/event.html

```diff
@@ -38,23 +38,7 @@ <h2 class="pull-left">
   {% if lat %}
     <div class="col-md-4">
       <div id="map"></div>
-      <script>
-        L.Icon.Default.imagePath = '/static/img/';
-        var map = L.map('map').setView([{{ lat }}, {{ long }}], 14);
-
-        L.tileLayer('http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
-          {
-            attribution: '&copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, <a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>',
-          }).addTo(map);
-
-        var popup = L.popup()
-          .setContent("<b>" + {{ _("Held At") | tojson }} + "</b><br> " + {{ place['name'] | tojson }});
-
-        var marker = L.marker([{{ lat }}, {{ long }}])
-          .addTo(map)
-          .bindPopup(popup)
-          .openPopup();
-      </script>
+      <script src="{{ 'edit.js'| make_static_path }}"></script>
```

Comment:
Be consistent. Either have spaces around `|` or don't, but stick to the same style.